README.md

```diff
@@ -86,6 +86,18 @@ Requesting: GET https://gitlab.com/api/v4/projects/666/search?scope=blobs&search
 Requesting: GET https://gitlab.com/api/v4/projects/999/search?scope=blobs&search=here-is-my-search-term
 ```
 
+## Installing from source
+1. Install node 14.x
+2. Clone this repository
+3. Build with
+   ```sh
+   $ npm run build
+   ```
+4. Run from bin directory
+   ```sh
+   $ bin/gitlab-search.js -h
+   ```
```

Comment:
Shouldn't this be a separate commit?

The prerequisites don't list a specific NodeJS version, and it doesn't seem to be required from what I tested.